server/server.go

```diff
@@ -25,7 +25,7 @@ func New() *Backend {
 func (b *Backend) AddUser(ctx context.Context, user *pbExample.User) (*types.Empty, error) {
 	b.mu.Lock()
 	defer b.mu.Unlock()
-	if user.GetCreateDate == nil {
```

Comment:
Maybe a unit test and a travis job would be good to also catch things like this?

Reply:
Good idea :). Could you make an issue?

Reply:
Done, but I think you can still merge this pull request.